extensions/adobe/experience/adcloud-experienceevent.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/analytics-experienceevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe AdCloud ExperienceEvent Base",
+  "type": "object",
+  "description":
+    "This AdCloud ExperienceEvent base schema includes the core/standard ExperienceEvent as well as the Adobe AdCloud ExperienceEvent Extensions.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/data/time-series",
+    "https://ns.adobe.com/xdm/context/identitymap",
+    "https://ns.adobe.com/xdm/context/experienceevent-advertising",
+    "https://ns.adobe.com/xdm/context/experienceevent-application",
+    "https://ns.adobe.com/xdm/context/experienceevent-channel",
+    "https://ns.adobe.com/xdm/context/experienceevent-commerce",
+    "https://ns.adobe.com/xdm/context/experienceevent-directmarketing",
+    "https://ns.adobe.com/xdm/context/experienceevent-environment-details",
+    "https://ns.adobe.com/xdm/context/experienceevent-marketing",
+    "https://ns.adobe.com/xdm/context/experienceevent-media",
+    "https://ns.adobe.com/xdm/context/experienceevent-profile-stitch",
+    "https://ns.adobe.com/xdm/context/experienceevent-search",
+    "https://ns.adobe.com/xdm/context/experienceevent-segmentmembership",
+    "https://ns.adobe.com/xdm/context/experienceevent-technical-details",
+    "https://ns.adobe.com/xdm/context/experienceevent-web",
+    "https://ns.adobe.com/experience/experienceevent",
+    "https://ns.adobe.com/experience/adcloud/experienceevent-all",
+    "https://ns.adobe.com/experience/target/experienceevent-shared",
+    "https://ns.adobe.com/experience/profile/experienceevent-shared",
+    "https://ns.adobe.com/experience/implementations-ext"
+  ],
+  "definitions": {
+    "adcloud-experienceevent": {
+      "properties": { }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/extensible#/definitions/@context"
+    },
+    {
+      "$ref": "https://ns.adobe.com/experience/experienceevent"
+    },
+    {
+      "$ref": "https://ns.adobe.com/experience/adcloud/experienceevent-all"
+    },
+    {
```

Comment:
I think the XC "shared" stuff should get mixed in once, in the experience namespace shared schema?